Gruntfile.coffee

```diff
@@ -113,16 +113,19 @@ module.exports = ->
             replacement: process.env.NOFLO_OAUTH_SERVICE_USER or 'https://api.flowhub.io'
           ,
             pattern: /\$NOFLO_OAUTH_CLIENT_ID/ig
-            replacement: process.env.NOFLO_OAUTH_CLIENT_ID or '9d963a3d-8b6f-42fe-bb36-6fccecd039af'
+            replacement: process.env.NOFLO_OAUTH_CLIENT_ID or '46fe25abef8d07e6dc2d'
+          ,
+            pattern: /\$NOFLO_OAUTH_CLIENT_REDIRECT/ig
+            replacement: process.env.NOFLO_OAUTH_CLIENT_REDIRECT or 'http://localhost:9999'
```

Comment:
Can you add a comment what this is for? Something like. `# Hosting location to redirect to after OAuth` ?